package.json

```diff
@@ -46,7 +47,8 @@
     "format-check": "prettier --check '**/*.{ts,tsx,js}'",
     "jest": "jest",
     "test": "yarn run jest && yarn run format-check && yarn run tslint",
-    "tslint": "tslint --project ./tsconfig.json"
+    "tslint": "tslint --project ./tsconfig.json",
+    "serve": "npm run build && clear && gatsby serve"
```

Comment:
Can `yarn` be used consistently instead of `npm` since the other scripts use `yarn` to execute?